jetty-server/src/test/java/org/eclipse/jetty/server/ServletRequestWrapperTest.java

```diff
@@ -0,0 +1,94 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2019 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.server;
+
+import org.eclipse.jetty.io.ByteArrayEndPoint;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.servlet.AsyncContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletRequestWrapper;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+public class ServletRequestWrapperTest
+{
+    private Server server;
+    private LocalConnector connector;
+    private RequestHandler handler;
+
+    @BeforeEach
+    public void init() throws Exception
+    {
+        server = new Server();
+        connector = new LocalConnector(server, new HttpConnectionFactory());
+        server.addConnector(connector);
+
+        handler = new RequestHandler();
+        server.setHandler(handler);
+        server.start();
+    }
+
+    @Test
+    public void servlet_request_wrapper()
```

Comment:
No snake case method names. It should also start with "test", as in "testServletRequestWrapper".

Reply:
well I'm not here to start a discussion on checkstyle rules as it can be long :P 
but method starting with test... was required in old junit now as we use annotation it's not required anymore. I find a bit redundant here to have both test in name: ServletRequestWrapperTest#testServletRequestWrapper but yes this can turn in a long discussion :)
by the way we do not activate checkstyle reporting for test sources should we?

Reply:
@olamy methods starting with "test" are not necessary, but it's far easier to find them in IDE by search and type rather than "search only those annotated with @Test" which IDE cannot do. So yes, old style and all, but so handy.

We tolerate old methods written in snake case, but there is no point for the new tests to not follow the rules that we have set for the source code.